src/gui/editors/AutomationEditor.cpp

```diff
@@ -680,6 +681,39 @@ void AutomationEditor::mouseReleaseEvent(QMouseEvent * mouseEvent )
 
 
 
+
+void AutomationEditor::removePoints( int x0, int x1 )
+{
+	int deltax = qAbs( x1 - x0 );
+	int x = x0;
+	int xstep;
+
+	if( deltax < AutomationPattern::quantization() )
+	{
+		return;
+	}
+
+	if( x0 < x1)
```

Comment:
This looks somewhat odd. Space after `(`, but no space before `)`.

Reply:
Fixed.